package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "lab-01-modular-patterns",
+  "version": "0.0.1",
+  "description": "lab-gnoevil",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "lint": "./node_modules/eslint/bin/eslint.js .",
+    "test": "mocha"
```

Comment:
Your test script should be the path name of the mocha folder in the Node modules, not "mocha". If someone else besides me were to pull down your file and test it, using that script makes it so a user doesn't have to install Mocha globally. 

Refer to the package.json in class-03-runtime-and-fs/lab-duncan and you can just copy/paste that pathname.